### PR TITLE
Internal: On "Create RC Tag" action, create backup branch (Core) [ED-10287]

### PR DESCRIPTION
**Pull request by maork-elementor**

### CODE REVIEW

File: .github/workflows/create-branch/action.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+        # Create a new branch from the specified branch, and push it to the remote repository
+        git branch $BRANCH_NAME ${{ inputs.SOURCE_BRANCH }}
+
+        # Switch to the new branch
+        git checkout $BRANCH_NAME
+        # Push the new branch to the remote repository and set the upstream branch to track it
+        git push -u origin $BRANCH_NAME
+
+
```
Comment (yotamselementor) [inline .github/workflows/create-branch/action.yml:29]:
```suggestion
```

File: .github/workflows/create-branch/action.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  SOURCE_BRANCH:
+    description: 'The source branch to create the new branch from (e.g. release/3.11.2)'
+    required: true
+
+runs:
+  using: "composite"
+  steps:
+    - shell: bash
+      run: |
+        #remove split on / and get the last part
```
Comment (yotamselementor) [inline .github/workflows/create-branch/action.yml:17]:
```suggestion
        # Remove split on / and get the last part
```

**Review by yotamselementor — CHANGES_REQUESTED** [review]

File: .github/workflows/create-branch/action.yml (hunk 2 above)
Comment (maork-elementor) [reply .github/workflows/create-branch/action.yml]: Done

File: .github/workflows/create-branch/action.yml (hunk 1 above)
Comment (maork-elementor) [reply .github/workflows/create-branch/action.yml]: Done

**Review by yotamselementor — APPROVED** [review]

**Pull request merged**